create_bot_knowledge.py

```diff
@@ -80,6 +80,52 @@ def connect_to_source(project: Tuple[str, str]):
     return service, repo
 
 
+def check_directory(knowledge_dir, update_knowledge: bool):
```

Comment:
We should also add the type to each parameter and the description under each function. I suggest you use `typing` library which is already in Python: https://docs.python.org/3/library/typing.html. As examples please take a look at any of Thoth repositories. For example: https://github.com/thoth-station/storages/blob/c3df96a97185753cc6a2e0b1318a4d391aa97499/thoth/storages/graph/postgres.py#L4817. So we have nice code.